report/Appendix/StandardLibrary.tex

```diff
@@ -0,0 +1,88 @@
+\section{Standard Library Functions}
+\label{standardLib}
+HCL includes multiple functions as part of its standard library.
+All standard library functions are implemented exclusively using the HCL syntax and builtin-functions. 
+
+%\textbf{Renamed Functions}\\
+%Functions stated in table \ref{tbl:renamed} are merely renamings of built-in functions for alternative naming. 
+%
+%\begin{table}[h!]
+%	\centering
+%	\caption{Renamed functions}
+%	\label{tbl:renamed}
+%	\begin{tabular}{|c|c|}
+%		\hline
+%		HCL Type & C++ Equivalent \\ \hline
+%		num      & double         \\ \hline
+%		txt      & string         \\ \hline
+%		bool     & bool           \\ \hline
+%		tuple    & struct         \\ \hline
+%		list     & list           \\ \hline
+%		func     & lambda         \\ \hline
+%		none     & void           \\ \hline
+%	\end{tabular}
+%\end{table}
+
+Snippet \ref{lis:st-to} shows the 'to' function used to create a range of numbers. Returns list of elements within range.
+\begin{lstlisting}[language=Kotlin,label=lis:st-to,caption=\texttt{to} function.]
+var to (num from, num to):list[num] {
+	num i = from
+	list[num] output
+	{
+		output = output + [i]
+		i = i + 1
+	} while { i < to }
+}
+\end{lstlisting}
+
+Snippet \ref{lis:st-any}, returns true if any element in a list uphold compare predicate. Returns false if no element in list upholds compare predicate.
```

Comment:
"upholds the predicate from compareFunc" lyder bedre.

Also, ikke newline efter punktum